dotcom-rendering/src/amp/components/elements/TwitterBlockComponent.tsx

```diff
@@ -61,14 +61,11 @@ const TextStyle = (pillar: ArticleTheme) => css`
 
 const makeFallback = (html: string): string | null => {
 	const { window } = new JSDOM(html);
-	const blockquotes = window.document.getElementsByTagName('blockquote');
+	const blockquote = window.document.getElementsByTagName('blockquote')[0];
 
-	if (blockquotes.length !== 1) {
-		return null;
-	}
```

Comment:
Did we use to return nothing if there was more than one blockquote? Confusing.

Reply:
Indeed, I'm not entirely sure why it was checking if the length was exactly 1. Looking at an example twitter embed it does look like there should be just the one blockquote.

```json
{
  "_type": "model.dotcomrendering.pageElements.TweetBlockElement",
  "html": "<blockquote class=\"twitter-tweet\"><p lang=\"en\" dir=\"ltr\">I did a mini-thread-within-a-thread here about a new data point in Ukraine&#39;s HIMARS attack on a reported Wagner base in Kadiivka - Pavel Prigozhin (Yevgeny&#39;s son) was reportedly photographed at the site. I haven&#39;t seen any news about him still being there when the attack happened <a href=\"https://t.co/on1stIL0Cw\">https://t.co/on1stIL0Cw</a></p>&mdash; Aric Toler (@AricToler) <a href=\"https://twitter.com/AricToler/status/1602158096075350018?ref_src=twsrc%5Etfw\">December 12, 2022</a></blockquote>\n",
  "url": "https://twitter.com/AricToler/status/1602158096075350018?s=20&t=lCXLzAqQZ-hiV4Giy5RKkA",
  "id": "1602158096075350018",
  "hasMedia": false,
  "role": "inline",
  "isThirdPartyTracking": false,
  "source": "Twitter",
  "elementId": "f71fbaf8-d108-41b9-a716-55d61a11d77b"
}
```

Reply:
I would not mind loosening this check from “exactly one” to “at least one”, but I'm not sure the ways in which it might fail, and am tempted to keep this PR as close as it can be to the original logic.